docs/ibc/introduction.mdx

```diff
@@ -0,0 +1,34 @@
+
+## How It Works: A Secure Messaging Protocol
+
+IBC ensures the authenticated and ordered delivery of data packets between chains. This is achieved through a lifecycle managed by the core protocol, on-chain light clients, and off-chain relayers.
+
+### Packet Creation and Commitment
+
+An application on a source chain initiates a cross-chain action by creating a Packet. This packet contains the application data, destination details, and an expiration timestamp (timeout). Instead of sending the packet directly, the source chain records a cryptographic commitment to this packet in its own ledger, creating a verifiable, tamper-proof record.
+
+### Relaying and Verification
+
+Independent, off-chain services called Relayers monitor chains for these commitments. A relayer fetches the committed packet data and a cryptographic proof of that commitment, then submits both to the destination chain. The destination chain uses its on-chain light client—an efficient module that tracks the state of the source chain—to verify the proof. This verification confirms the packet's authenticity and origin without trusting the relayer.
+
+### Delivery and Acknowledgement
+
+Once a packet is verified, its data is delivered to the destination application for processing. The destination chain then commits an Acknowledgement, recording the outcome of the transaction (either success or failure). This acknowledgement acts as a return receipt, which a relayer then delivers back to the source chain along with its own proof.
+
+### Finalization and Atomic Settlement
+
+The source chain verifies the acknowledgement, providing the original application with definitive proof of the transaction's outcome. This allows the application to finalize its state (e.g., by releasing escrowed assets). If a packet is not acknowledged before its timeout, the protocol ensures the original transaction can be safely reverted. This guarantees atomic settlement: the entire cross-chain operation either successfully completes on both chains or is cleanly rolled back, preventing funds from being lost or stuck.
+
+## Architectural Benefits
+
+- **Protocol Standardization:** By defining a single, open standard for communication, IBC eliminates the need for applications to implement and maintain multiple, custom bridges. This reduces engineering overhead and contains systemic risk by avoiding a fragmented landscape of disparate, unaudited protocols.
+
+- **Host-Chain Security Model:** IBC's security model is based on the consensus of the connected chains. It does not introduce a separate, trusted third-party or multisig validator set, which are common points of failure in other interoperability solutions. The security of a transaction is therefore equivalent to the security of the participating state machines.
```

Comment:
This is the light client model, but with the attestation model, we are actually introducing the concept of a trusted party/multisig as a way to do this 😅